core/src/telemetry/otlp.rs

```diff
@@ -192,8 +193,8 @@ fn init_counters(
 	meter: Meter,
 	origin: &Origin,
 	project_name: String,
-) -> HashMap<&'static str, Counter<u64>> {
-	[
+) -> Result<HashMap<&'static str, Counter<u64>>> {
+	let counters = [
```

Comment:
Is there any reason why this function now return `Result`?

Reply:
But only possible result here is OK? There is no error branch if I didn't missed something.

Reply:
no reason to return Result, yet